### PR TITLE
Update psycopg2 version

### LINKED ISSUE #91: Update psycopg2 to version 2.9.9
As mentioned in this comment https://github.com/fossasia/eventyay-tickets/pull/85#pullrequestreview-2060498875 we need the update of psycopg2 to version 2.9.9

### PR DESCRIPTION
**Pull request by progmatic-99**
Fixes #91

### CODE REVIEW

File: .github/workflows/tests.yml
Comment (norbusan) [inline .github/workflows/tests.yml]: You don't need to add a version to psyscopg2-binary since it is already pinned in `src/requirements/dev.txt` below

File: src/setup.py
Comment (norbusan) [inline src/setup.py]: Why do you include unrelated changes here?

File: src/setup.py
Comment (norbusan) [inline src/setup.py]: Why do you include unrelated changes here?

File: src/setup.py
Comment (norbusan) [inline src/setup.py]: All the above ... Why do you include unrelated changes here?

File: src/setup.py
Comment (norbusan) [inline src/setup.py]: Why do you include unrelated changes here?

File: src/setup.py
Comment (norbusan) [inline src/setup.py]: Why do you include unrelated changes here?

File: src/setup.py
Comment (norbusan) [inline src/setup.py]: Why do you include unrelated changes here?

**Review by norbusan — CHANGES_REQUESTED** [review]
I don't like PRs that change a lot of unrelated changes.

Make the change of quotes a separate PR if you want, and keep the psycopg2 version change minimal. Thanks.

File: src/setup.py
Comment (progmatic-99) [reply src/setup.py]: my flaker must have kicked in when i saved the file, should i revert this?

File: src/setup.py
Comment (progmatic-99) [reply src/setup.py]: same as above

File: src/setup.py
Comment (mariobehling) [reply src/setup.py]: yes, please revert unrelated changes.

Comment (norbusan) [reply src/setup.py]: Yes, please revert all unrelated changes. Thanks

**Review by norbusan — COMMENTED** [review]
Did you test that the program actually runs with this version?

**Review by marcoag — CHANGES_REQUESTED** [review]
If psycopg2 gets upgraded to more than the current version the database connection throws an issue with "database not set to UTC", so it will fail. Please do not merge this unless you've tested this locally and have a workaround.

The issue is commented here: https://stackoverflow.com/questions/38807296/django-1-9-2-assertionerror-database-connection-isnt-set-to-utc#39649018

**Comment by progmatic-99** [comment]
@marcoag I ran the project with the updated psycopg2 version on my local and also created an event. No problems with db at my end

**Comment by marcoag** [comment]
@progmatic-99 are you sure? I just retested your branch with a full deployment and as migrations fail with:
```
File "/usr/local/lib/python3.8/site-packages/django/db/backends/postgresql/utils.py", line 6, in utc_tzinfo_factory
    raise AssertionError("database connection isn't set to UTC")
AssertionError: database connection isn't set to UTC
```

Can you please check your container has the correct `psycopg2` version and test?
```
pretixuser@9a06140c7fd3:~/src$ python 
Python 3.8.19 (default, Apr 10 2024, 15:23:25) 
[GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import psycopg2
>>> psycopg2.__version__
'2.9.9 (dt dec pq3 ext lo64)'
```

**Comment by progmatic-99** [comment]
@marcoag this is compatible with python 3.9
image

**Pull request closed without merging**
